pkg/controller/service/service_controller.go

```diff
@@ -638,20 +575,25 @@ func (s *ServiceController) nodeSyncLoop() {
 	if nodeSlicesEqualForLB(newHosts, s.knownHosts) {
 		// The set of nodes in the cluster hasn't changed, but we can retry
 		// updating any services that we failed to update last time around.
-		s.servicesToUpdate = s.updateLoadBalancerHosts(s.servicesToUpdate, newHosts)
+		// TODO(horwitz): the return of this isn't useful... change the api??
+		s.updateLoadBalancerHosts(s.servicesToUpdate, newHosts)
```

Comment:
Anyone know the purpose of `s.servicesToUpdate` ? It wasn't actually used any where but it was set?